### PR TITLE
Explain how to run terragrunt destroy

### LINKED ISSUE #395: [AWS] codelab terragrunt destroy doesn't appear to work
**Description**

Having finished playing with my AWS codelab instance, I wanted to turn it down and release resources, so I ran `terragrunt destroy` from the `deployment/live/aws/codelab` directory.

The result was ... nothing happened. The command just hangs with no output.

Running `terragrunt plan` or `terragrunt apply` from the same directory got me a bunch of output saying that everything was hunky dory and that resources matched config, so whatever it is appears only to affect the `destroy` command.

I noticed that if I hit <return> the hanging command does exit after a little while.

### PR DESCRIPTION
**Pull request by phbnf**
Fixes #395

### CODE REVIEW

File: deployment/live/aws/codelab/README.md [hunk 1]
```diff
@@ -163,4 +163,9 @@ endpoint.
 
 > [!IMPORTANT]  
 > Do not forget to delete all the resources to avoid incuring any further cost
-> when you're done using the log. The easiest way to do this, is to [close the account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html).
\ No newline at end of file
+> when you're done using the log. The easiest way to do this, is to [close the account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html).
+> If you prefer to delete the resources with `terragrunt destroy`, bare in mind
+> that this command might not destroy all the resources that were created (like
+> the S3 bucket or DynamoDB instance Terraform created to store its state for
+> instance). If `terragrunt destroy`  shows no putput, run
```
Comment (roger2hk) [inline deployment/live/aws/codelab/README.md:170]:
```suggestion
> instance). If `terragrunt destroy` shows no putput, run
```

**Review by roger2hk — APPROVED** [review]

File: deployment/live/aws/codelab/README.md [hunk 2]
```diff
@@ -163,4 +163,9 @@ endpoint.
 
 > [!IMPORTANT]  
 > Do not forget to delete all the resources to avoid incuring any further cost
-> when you're done using the log. The easiest way to do this, is to [close the account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html).
\ No newline at end of file
+> when you're done using the log. The easiest way to do this, is to [close the account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html).
+> If you prefer to delete the resources with `terragrunt destroy`, bare in mind
```
Comment (AlCutter) [inline deployment/live/aws/codelab/README.md:167]:
```suggestion
> If you prefer to delete the resources with `terragrunt destroy`, bear in mind
```

File: deployment/live/aws/codelab/README.md (hunk 1 above)
Comment (AlCutter) [inline deployment/live/aws/codelab/README.md:170]:
```suggestion
> instance). If `terragrunt destroy`  shows no output, run
```

**Review by AlCutter — APPROVED** [review]

File: deployment/live/aws/codelab/README.md (hunk 1 above)
Comment (mhutchinson) [reply deployment/live/aws/codelab/README.md:170]: Can you fix the typo on `output` too?

**Pull request merged**